static/docker-compose.yml

```diff
@@ -91,18 +127,8 @@ volumes:
   dbdata: {}
   uploads: {}
   letsencrypt: {}
-  redis:
-    driver: local
-    driver_opts:
-      type: 'none'
-      o: 'bind'
-      device: '${PLEXTRAC_HOME:-.}/volumes/redis'
-  couchbase-backups:
-    driver: local
-    driver_opts:
-      type: 'none'
-      o: 'bind'
-      device: '${PLEXTRAC_HOME:-.}/backups/couchbase'
+  redis: {}
```

Comment:
The bind mounts were causing docker-compose to fail while creating the volumes inside the Vagrant box (inside my MacOS host machine).  We may have hit the inception limit here.

It doesn't look like we're using the named volumes anyway and this change fixes the problem.

Reply:
Is this going to break anything on the production Linux hosts? 🤔  cc @jesseops

Reply:
Yes, we definitely don't want to drop these as we'll move into undefined territory. @evanhsu can you post error logs? Within vagrant this _shouldn't_ behave any differently between host machines since it's running in a VM either way.

Reply:
I removed the changes to the named volumes from this PR and I'll post some error logs shortly once my vagrant box rebuilds

Reply:
In an unsurprising turn of events, rebooting my laptop twice appears to have resolved this issue 🤦🏻